frontend/src/scenes/funnels/FunnelViz.js

```diff
@@ -96,7 +97,15 @@ export function FunnelViz({
         return steps && steps.length > 0 && steps[0].labels ? (
             <>
                 <div style={{ position: 'absolute', marginTop: -20, textAlign: 'center', width: '90%' }}>
-                    % of users converted between first and last step
+                    converted within&nbsp;
+                    <InputNumber
```

Comment:
Last thing I remembered is that this is only supported on ee right? So you'll need to put this behind the `preflight?.is_clickhouse_enabled` condition